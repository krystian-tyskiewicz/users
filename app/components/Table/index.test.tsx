import { render } from '@testing-library/react'
import { Table, TBody, THead, Tr, Th, Td } from '.'

it('renders correctly', () => {
  const { container } = render(
    <Table>
      <THead>
        <Tr>
          <Th>Id</Th>
          <Th>Name</Th>
        </Tr>
      </THead>
      <TBody>
        <Tr>
          <Td>1</Td>
          <Td>John Doe</Td>
        </Tr>
      </TBody>
    </Table>
  )
  expect(container).toMatchSnapshot()
})
