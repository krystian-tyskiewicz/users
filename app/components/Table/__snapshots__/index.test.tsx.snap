// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <table
    class="sc-bdvumH cPURWe"
  >
    <thead
      class="sc-gsDKeI"
    >
      <tr
        class="sc-hKwBBi"
      >
        <th
          class="sc-eCIlHX ctgWaU"
        >
          Id
        </th>
        <th
          class="sc-eCIlHX ctgWaU"
        >
          Name
        </th>
      </tr>
    </thead>
    <tbody
      class="sc-dkPulx"
    >
      <tr
        class="sc-hKwBBi"
      >
        <td
          class="sc-jRQBzY gSFLqd"
        >
          1
        </td>
        <td
          class="sc-jRQBzY gSFLqd"
        >
          John Doe
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
